examples/cryptorealtime/src/main/java/utils/ExchangeConfiguration.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Ivo Galic https://github.com/galic1987
+ * <p/>
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is this a brand new file you wrote from scratch, or are you modifying an existing Apache project's file?

Reply:
90% is of the code is new. should I change something in files where I reused the snippets from other libraries?